zhmcclient/_user.py

```diff
@@ -275,6 +277,8 @@ def update_properties(self, properties):
         # HTTPError to be raised in the POST above, so we assert that here,
         # because we omit the extra code for handling name updates:
         assert self.manager._name_prop not in properties
+        if properties is not None:
+            properties.pop('password', None)
```

Comment:
I don't think the removal of the "password" property should be done from the `properties` input parameter, because of the side effect of changing the input parameter.

I suggest to remove the "password" property from the copy of the `properties` input parameter which is created one line down.

Also, there should be an entry in the change log about this change. It is an incompatible change since up to now, users were able to store the password in the User object and now that no longer works.

Reply:
Done